readthedocs/templates/core/project_details.html

```diff
@@ -19,40 +19,42 @@
     <h3>{% trans "Versions" %}</h3>
   </div>
 
-  <div class="module-list">
```

Comment:
I think this div is still needed, can you post a screenshot of how it looks?

Reply:
![tests-demo _ read the docs](https://user-images.githubusercontent.com/14057588/36638052-52b7b0be-1a0f-11e8-9f67-b9f7f892af7a.png)
Because of removing that div it looks like the version page listing.

Reply:
Actually, rtd is moving to match the previous style on all lists (see https://github.com/rtfd/readthedocs.org/pull/3445#issuecomment-366299122)

Reply:
But the issue #3554 here says it should match the style on project version listing page. 
What should be done?

Reply:
I'm not sure, lets wait to agjohnson's answer, but on other issue this was already discussed (see #3572, relevant comments: https://github.com/rtfd/readthedocs.org/pull/3572#issuecomment-364497418, https://github.com/rtfd/readthedocs.org/pull/3572#issuecomment-367514230)

Reply:
Yes. I saw that. I still went ahead with this issue as there were other improvements as well. 
I hope that's fine. :)

Reply:
The styling that was removed here was indeed the correct styling, it should be added back. The styling wasn't the issue, but the table content. We're also missing an opportunity to have a single template for 3 different pages, so that is the larger focus here.

Reply:
Oh! In that case, the previous design should remain same. Got it! I'll change the styling back again and keep the table content modified as it is in PR.